uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -865,4 +865,17 @@ public function testIsCalculationUnitInHoursReturnsFalseWhenCalculationUnitIsNot
 
     $this->assertFalse($absenceType->isCalculationUnitInHours());
   }
+
+  public function testGetCategories() {
+    $categoryOptions = [
+      '1' => 'Leave',
+      '2' => 'Sickness',
+      '3' => 'TOIL',
+      '4' => 'Custom'
+    ];
+    $categories = AbsenceType::getCategories();
+
+    $this->assertCount(4, $categories);
```

Comment:
On a side note, we don't really need this assertion, checking that both arrays are equal is enough.